engines/pyfunc-ensembler-service/pyfunc_ensembler_runner/ensembler_runner.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, artifact_dir: str):
     def load(self):
         self._ensembler = pyfunc.load_model(self.artifact_dir)
 
-    def predict(self, inputs: Dict[str, Any]) -> List[Any]:
-        logging.info(f"Input request payload: {inputs}")
-        output = self._ensembler.predict(inputs)
+    def predict(self, body: Dict[str, Any], headers: Dict[str, str]) -> List[Any]:
```

Comment:
This call's MLFlow Pyfunc model's predict which is eventually received [here](https://github.com/gojek/turing/blob/main/sdk/turing/ensembler.py#L62) at runtime. The `self.ensemble` call from  `_ensemble_request` finally calls the implementation provided by the user.